chainer/link.py

```diff
@@ -578,6 +578,26 @@ def serialize(self, serializer):
         for name in self._persistent:
             d[name] = serializer(name, d[name])
 
+    def count_params(self):
+        """Count the total size of parameters.
+
+        If the link containts uninitialized parameters, this raises warnings.
```

Comment:
I think we should emphasize that this method counts the number of scalars.
How about something like this?
(Just an example. Please fix as you like)

*Counts the total number of parameters.*

*This method counts the total number of scalar values included in all the `Parameter`s held by this link and its descendants.*

Reply:
That's right. Good suggestion